common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/Work.scala

```diff
@@ -113,21 +112,55 @@ sealed trait WorkState {
 
 object WorkState {
 
-  // TODO: for now just 2 states, in the end the states will correspond to the
-  // block comment above
-
-  case class Unidentified(
+  case class Source(
```

Comment:
Are there any other "interesting" files to look at beside this one? ie ones that aren't just renaming etc

Reply:
This is the main one, but probably also important to pay attention to what's changed in the merger (it now takes in `Work[Source]` and outputs `Work[Merged]`)